integration/sql/impl/src/visitor.rs

```diff
@@ -587,34 +617,58 @@ impl Visit for Statement {
                     expr.visit(context)?;
                 }
             }
-            Statement::AlterTable { name, operation } => {
-                match operation {
-                    AlterTableOperation::SwapWith { table_name } => {
-                        // both table names are inputs and outputs of the swap operation
-                        context.add_input(table_name.clone().0);
-                        context.add_input(name.clone().0);
-
-                        context.add_output(table_name.clone().0);
-                        context.add_output(name.clone().0);
-                    }
-                    AlterTableOperation::RenameTable { table_name } => {
-                        context.add_input(name.clone().0);
-                        context.add_output(table_name.clone().0);
+            Statement::AlterTable {
+                name,
+                if_exists: _,
+                only: _,
+                operations,
+                location: _,
+            } => {
+                for operation in operations {
+                    match operation {
+                        AlterTableOperation::SwapWith { table_name } => {
+                            // both table names are inputs and outputs of the swap operation
+                            context.add_input(table_name.clone().0);
+                            context.add_input(name.clone().0);
+
+                            context.add_output(table_name.clone().0);
+                            context.add_output(name.clone().0);
+                        }
+                        AlterTableOperation::RenameTable { table_name } => {
+                            context.add_input(name.clone().0);
+                            context.add_output(table_name.clone().0);
+                        }
+                        _ => context.add_output(name.clone().0),
                     }
-                    _ => context.add_output(name.clone().0),
                 }
             }
             Statement::Delete {
-                table_name,
+                tables: _,
+                from,
                 using,
                 selection,
                 ..
             } => {
-                let table_name = get_table_name_from_table_factor(table_name)?;
-                context.add_output(table_name);
+                match from {
+                    FromTable::WithFromKeyword(tables) | FromTable::WithoutKeyword(tables) => {
```

Comment:
would lobe to see new test in `tests_delete.rs` to see diff between `WithFromKeyword`  and `WithoutKeyword` as well as deleting from multiple tables.

Reply:
Added test.